tests/tests/parser/structs/comma_separators.leo

```diff
@@ -0,0 +1,14 @@
+/*
+namespace: Parse
```

Comment:
Nit. Append `_fail` to the file name of this test case as it is expected to fail.